### PR TITLE
Remove the workaround to put the IBC files in the right directory

### PR DESCRIPTION
**Pull request by adiaaida**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -46,6 +46,6 @@
     <!-- Standard dependencies -->
     <NETStandardLibraryPackageVersion>2.1.0-prerelease.19158.1</NETStandardLibraryPackageVersion>
     <!-- dotnet-optimization dependencies -->
-    <optimizationwindowsntx64IBCCoreFxPackageVersion>99.99.99-master-20190227.1</optimizationwindowsntx64IBCCoreFxPackageVersion>
+    <optimizationwindows_ntx64IBCCoreFxPackageVersion>99.99.99-master-20190308.5</optimizationwindowsntx64IBCCoreFxPackageVersion>
```
Comment (ViktorHofer) [inline eng/Versions.props]: Was renaming the property intentional? If yes you also need to change the closing element.

Comment (adiaaida) [reply eng/Versions.props]: Yes. Maestro hasn't been updating the package version because this was incorrect. Will fix. Thanks

**Comment by ericstj** [comment]
@BruceForstall this will avoid the max path issue you are hitting in CoreCLR.

**Comment by BruceForstall** [comment]
Is this ready to merge?

**Comment by ericstj** [comment]
Ubuntu was down yesterday.  I'm OK merging this.  It doesn't have platform specific test behavior.

**Pull request merged**